### PR TITLE
Set Up Sat 6.2 REPOS and REPOSET

### LINKED ISSUE #3757: Rolling Back REPOS and REPOSET on 6.2.z branch
This was discussed on weekly sprint mtg

### PR DESCRIPTION
**Pull request by renzon**
close #3757

6.2.z branch must contain only 6.2 sat repos

### CODE REVIEW

**Comment by sghai** [comment]
ACK.

Please note that this change is against `6.2.z` branch and supports Sat 6.2 version specifically for tools repos.

@oshtaier @abalakh @kbidarkar @lpramuk @rplevka : Please 'ACK'  this change once you'r done with review.

**Comment by kbidarkar** [comment]
ACK

File: robottelo/constants.py [hunk 1]
```diff
@@ -234,21 +240,27 @@
...
     ('rhel', 'rhva6', 'rhva6S', 'repo_arch', 'x86_64'),
     ('rhel', 'rhva6', 'rhva6S', 'repo_ver', '6Server')
 ]
 
 SAT6_TOOLS_TREE = [
-    ('rhel', 'rhst6', 'rhst6', 'repo_name',
-     'Red Hat Satellite Tools 6.1 for RHEL 6 Server RPMs x86_64'),
+    (
+        'rhel', 'rhst6', 'rhst6', 'repo_name',
+        'Red Hat Satellite Tools 6.1 for RHEL 6 Server RPMs x86_64'
```
Comment (rplevka) [inline robottelo/constants.py]: did you mean `6.2`?

File: robottelo/constants.py [hunk 2]
```diff
@@ -234,21 +240,27 @@
...
+    ),
     ('rhel', 'rhst6', 'rhst6', 'repo_arch', 'x86_64'),
     ('rhel', 'rhst6', 'rhst6', 'repo_ver', '6.1'),
```
Comment (rplevka) [inline robottelo/constants.py]: did you mean `6.2`?

**Comment by rplevka** [comment]
Overall, there are 5 occurrences of `6.1` in this file. Could you confirm it is not supposed to be changed to `6.2`?
ACK pending comments.

**Comment by renzon** [comment]
@rplevka: I was supposed to change only REPOS and REPOSET. @sghai could you confirm it?

**Comment by sghai** [comment]
@renzon @rplevka : `SAT6_TOOLS_TREE` is being used by test `test_positive_puppet_install` [test ](https://github.com/SatelliteQE/robottelo/blob/master/tests/foreman/endtoend/test_ui_endtoend.py#L343).

Let's change  this too with latest released version of sat tools i.e 6.2

**Comment by sghai** [comment]
ACK

**Pull request merged**